src/main/assembly/assembly.xml

```diff
@@ -0,0 +1,38 @@
+<!--
+ Describes the plugin archive
+
+ @author Decebal Suiu
+ @version 1.0
+-->
+<assembly>
+	<id>plugin</id>
+	<formats>
+		<format>zip</format>
+	</formats>
+	<includeBaseDirectory>false</includeBaseDirectory>
+	<dependencySets>
+		<dependencySet>
+            <useProjectArtifact>false</useProjectArtifact>
+            <scope>runtime</scope>
+			<outputDirectory>lib</outputDirectory>
+			<includes>
+				<include>*:jar:*</include>
+			</includes>
+		</dependencySet>
+	</dependencySets>
+    <!--
```

Comment:
As Denis suggested in another PR, best not to have commented out code like this unless you expect to uncomment it.

Reply:
removed comment

Reply:
Yes I tested it with the built zip file